R/adjust_daily_incidence.R

```diff
@@ -0,0 +1,78 @@
+#' Redistribute difference between the original observation and the
+#' imputed incidence value for the single day with reporting anomaly
+#'
+#' @param data time series data for location in adjustment case
+#' @param adjustment_case a case to adjust
```

Comment:
Can we add a little more detail describing this parameter?  It looks like the function is expecting this to be a list of length one with one named element, "date"?  If that's true, could we just make this the `adjustment_date`?